src/Confluent.Kafka/Consumer.cs

```diff
@@ -284,10 +315,12 @@ public void Assign(TopicPartition partition)
 
 
         /// <summary>
-        ///     Update the assignment set to a single <paramref name="partition" />.
+        ///     Update the set of assigned partitions (the set of partitions the
```

Comment:
dito , both for wording and neccesity

Reply:
updated the wording. I do think this overload is useful enough to keep (in particular, it's very useful for the single partition topic case). It's used 31 times in the integration tests, and I use it myself in real code in practice.